src/Omega_h_vector.hpp

```diff
@@ -150,6 +150,16 @@ OMEGA_H_INLINE Vector<n> zero_vector() OMEGA_H_NOEXCEPT {
   return fill_vector<n>(0.0);
 }
 
+template <Int n>
+OMEGA_H_INLINE Vector<n> max_float_vector() OMEGA_H_NOEXCEPT {
```

Comment:
It looks like the Omega_h `Vector` type only supports storing `Omega_h::Real` values.  Given that, I think we should remove the `_float` from the function names.   This also applies to `min_float_vector`.